README.md

```diff
@@ -1 +1,29 @@
-# Serializer
\ No newline at end of file
+# WIP: Cerializer
```

Comment:
Could you write down some simple schema and show the serialization/deserialization code it produces so user can imagine the way it is done? I think that you don't mention the Cython part anywhere.